cmd/root/postgres/dbinfo.go

```diff
@@ -210,7 +210,7 @@ func (i *DBInfo) fetchDBInstance(ctx context.Context) error {
 	sqlInstance := sqlInstances.Items[0]
 
 	connectionName, ok, err := unstructured.NestedString(sqlInstance.Object, "status", "connectionName")
-	if !ok {
+	if !ok || err != nil {
```

Comment:
Burde vi ikke skilt mellom disse to? For nå blir ikke `err` skrevet ut, og den kan vel være knyttet til noe annet enn feilmeldingen vi skriver?

Reply:
Var noe jeg la til fordi den aldri var sjekket (har nytt lint oppsett :P ) Men tror i utgangspunktet aldri `err` ender opp med å bli satt uten at det har vært error tidligere i koden